src/App.js

```diff
@@ -1,13 +1,26 @@
-import { BrowserRouter as Router } from 'react-router-dom';
+import { Routes, Route } from 'react-router-dom';
+import { useSelector } from 'react-redux';
 import Navigation from './layout/navigation/Sidebar';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import MyReservationsPage from './components/reservations/MyReservationsPage';
 
 function App() {
+  const currentUser = useSelector((state) => state.auth);
+
   return (
-    <div className="App">
-      <Router>
-        <Navigation />
-      </Router>
+    <div className="App d-flex">
+      <Navigation />
+      <Routes>
+        <Route
+          exact="true"
+          path="/reservations"
+          element={
+          currentUser.token && currentUser.token !== null
```

Comment:
checking for currentUser.token is enough, no need for rechecking with currentUser.token !== null

Reply:
I believe it is necessary because the token is already set to null. So any change from null means a user is present.